JavaScript/2-benchmark.js

```diff
@@ -3,15 +3,16 @@
 const benchmark = {};
 module.exports = benchmark;
 
-benchmark.do = (count, retry, tests) => {
+benchmark.do = (count, retry, tests, namesOfTests) => {
 
   let k;
   for (k = 0; k < retry; k++) {
-    tests.map(test);
+    let counter = 0; // Order number of function that will be call
```

Comment:
I prefer single-line comments to be separated from code with two spaces, but I believe it is not required by our code style, so you may ignore this :)